addons/crm_iap_enrich/models/crm_lead.py

```diff
@@ -41,6 +37,15 @@ def _iap_enrich_leads_cron(self):
         ])
         leads.iap_enrich(from_cron=True)
 
+    @api.model_create_multi
+    def create(self, vals_list):
+        leads = super(Lead, self).create(vals_list)
+        enrich_mode = self.env['ir.config_parameter'].sudo().get_param('crm.iap.lead.enrich.setting', 'manual')
+        if enrich_mode == 'auto':
+            cron = self.env.ref('crm_iap_enrich.ir_cron_lead_enrichment')
+            cron._trigger(fields.Datetime.now())
```

Comment:
Don't think you have to pass "fields.Datetime.now()", I think the method does that if you don't pass it anything.

Reply:
Hello @awa-odoo 

Changes done.

Thanks